Digipost/src/main/res/values/styles.xml

```diff
@@ -3,15 +3,17 @@
 
     <style name="Digipost" parent="Theme.AppCompat.Light.DarkActionBar">
```

Comment:
Her har noen farger blitt erstattet med farger fra styleguide + default-farger for tekst i app og i menyer er satt